testing/trino-product-tests/src/main/java/io/trino/tests/hive/TestHiveBasicTableStatistics.java

```diff
@@ -176,23 +175,27 @@ public void testCreatePartitioned()
         }
     }
 
-    @Test(groups = SKIP_ON_CDH /* CDH 5 metastore automatically gathers raw data size statistics on its own */)
-    public void testAnalyzePartitioned()
+    @Test(groups = SKIP_ON_CDH, dataProvider = "transactionalNonTransactionalProvider" /* CDH 5 metastore automatically gathers raw data size statistics on its own */)
```

Comment:
the comment floated away from the code piece it's commenting
```suggestion
    @Test(groups = SKIP_ON_CDH, // CDH 5 metastore automatically gathers raw data size statistics on its own 
            dataProvider = "transactionalNonTransactionalProvider")
```

Reply:
Changed.